### DISCUSSION TITLE
App Router Support for Faust blocks

### DISCUSSION THREAD

**Discussion by FosterChad95** [wpengine/faustjs#1819] [Q&A] [answered]
Hello!

My team is very interested in using Faust blocks for our next project. I have seen the RFC but was wondering if there was any changes possibly coming soon to support Faust blocks in the next.js app router? 

Thanks in advance!

**Comment by blakewilson** [comment, marked as answer]
Hi @FosterChad95,

We do not have any immediate plans to start work on supporting Faust blocks in our App Router support yet. This is in part due to our `experimental-app-router` package still being experimental, along with some of our dependencies like `@apollo/experimental-nextjs-app-support`.

We will likely start the planning for that once Apollo takes their app router support out of experimental.

**Reply by FosterChad95** [reply]
Great, thank you @blakewilson for the response!
